package.json

```diff
@@ -58,6 +60,7 @@
     ]
   },
   "resolutions": {
+    "blockprotocol": "0.0.7",
```

Comment:
Latest `block-scripts` and `block-template` use 0.0.8. I tried upgrading but failed generating a patch file, so decided to postpone this. Using 0.0.7 everywhere as before for now.